packages/react-components/src/InputAddress/index.tsx

```diff
@@ -80,6 +85,28 @@ function transformToAccountId (value: string): string | null {
     : accountId;
 }
 
+async function transformOrResolveToAccountId (value: string, { api, systemChain }: {api: ApiPromise, systemChain: string}): Promise<string | null> {
```

Comment:
Would be helpful to state _what_ is transformed. It'd be
```suggestion
async function transformOrResolveToAccountId (sthOrAzeroDomain: string, { api, systemChain }: {api: ApiPromise, systemChain: string}): Promise<string | null> {
```

Unfortunatelly I can't figure out what that `sth` would be without going very deep in the code.

Reply:
I think it's address, at least in all cases I checked